bbcu/libs/helpers.js

```diff
@@ -417,6 +417,7 @@ class InboundNonceWatcher {
 		this.stopped = false;
 
 		let attempts = 0;
+		let step_number = 1;
```

Comment:
Added this to keep track of what step we're on, 1 = connection, 4 = verification. This is what we'll key off instead of type for errors and logging. Doesn't support Credential yet, but it looks like we don't even use this class with that type anywhere.